with-nextjs/.env.example

```diff
@@ -0,0 +1,9 @@
+# create an organization access token from your organization settings in Polar.
+NEXT_PUBLIC_POLAR_ACCESS_TOKEN=
+# webhook
+POLAR_WEBHOOK_SECRET=
+
+# client url
```

Comment:
Please mention that this is the URL the customer would be led to if they purchase something.